packages/svelte-query/package.json

```diff
@@ -18,7 +18,7 @@
     "build": "svelte-kit sync && svelte-package && rimraf ./build/lib/package.json",
     "check": "svelte-kit sync && svelte-check --tsconfig ./tsconfig.json",
     "test:eslint": "svelte-kit sync && ../../node_modules/.bin/eslint --ext .svelte,.ts ./src",
-    "test:jest": "svelte-kit sync && vitest run --coverage"
+    "test:jest": "npm run check && vitest run --coverage"
```

Comment:
I've restructured the testing scripts here so this isn't needed: https://github.com/TanStack/query/pull/4779